packages/react/src/components/Modal/Modal.js

```diff
@@ -168,7 +168,7 @@ export default class Modal extends Component {
     modalLabel: '',
     selectorPrimaryFocus: '[data-modal-primary-focus]',
     focusTrap: true,
-    hasScrollingContent: true,
+    hasScrollingContent: false,
```

Comment:
This is a breaking change - Any more details wrt why you thought we need this change?

Reply:
I think actually having it set to `true` was the breaking change because existing code would have to be updated because of `aria-label` now needing to be defined and there is no default value for it specified for this component.

Reply:
just for some context the default value was initially decided in https://github.com/carbon-design-system/carbon/pull/3943#issuecomment-533508312, but since we now have a gradient to indicate scrolling content I guess we will default back to `hasScrollingContent = false`?

Reply:
@emyarod yeah, I think things got confused in the long thread. I think the reason we can't default to true is that it would break existing usage (as they don't specify aria-label). But if they include the prop, aria-label should be set. I don't think this is included in a stable version yet so can revert safely but will defer to you all in case it has been included 👍